nservicebus/messaging/databus/index_serialization_core_(,8.0).partial.md

```diff
@@ -0,0 +1,13 @@
+## Serialization
+
+By default NServiceBus uses a `BinaryFormatter` to serialize and deserialize data bus properties.
+
+WARN: `BinaryFormatter` [is not supported in .NET 5](https://docs.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/5.0/binaryformatter-serialization-obsolete). For projects that target .NET 5, use a custom serializer as described below.
+
+### Using a custom serializer
```

Comment:
I would put this in a separate document underneath  data bus,

Reply:
The serialization choice is mandatory now in v8. Separating it means users may miss it. I thought about separating the AdditionalDeserialization support to a separate page, but that's small and only for v8, so seems excessive.